test/FunctionalTests/Client/UnaryTests.cs

```diff
@@ -68,6 +68,35 @@ Task<HelloReply> UnaryThrowError(HelloRequest request, ServerCallContext context
         StringAssert.StartsWith("Failed to deserialize response message.", call.GetStatus().Detail);
     }
 
+    [TestCase("fr", "fr")]
+    [TestCase(null, "en-US")]
+    public async Task Unary_SetAcceptLanguage_ServerCultureChanged(string clientAcceptLanguage, string expectedServerCulture)
+    {
+        string? serverCulture = null;
+        Task<HelloReply> UnaryThrowError(HelloRequest request, ServerCallContext context)
+        {
+            serverCulture = Thread.CurrentThread.CurrentCulture.Name;
+            return Task.FromResult(new HelloReply { Message = serverCulture });
+        }
+
+        // Arrange
+        var method = Fixture.DynamicGrpc.AddUnaryMethod<HelloRequest, HelloReply>(UnaryThrowError);
+        var channel = CreateChannel();
+        var client = TestClientFactory.Create(channel, method);
+        var metadata = new Metadata();
+        if (clientAcceptLanguage != null)
+        {
+            metadata.Add("accept-language", clientAcceptLanguage);
```

Comment:
API thought unrelated to this PR, but more to the feature; should this be / is this already automated at the client? `metadata.AddRequestLocalization()`? an interceptor? (not sure either works perfectly; just asking the question)

Reply:
No other gRPC clients set this automatically.